src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -666,6 +666,43 @@ object TezosDatabaseOperations extends LazyLogging {
     }
   }
 
+  /** Updates an account by id, marking it with a flag.
+    * @param id the account identifier on database (corresponds to the pkh)
+    * @return the action to run against the db, which will tell how many rows where touched
+    */
+  def activateAccount(id: String): DBIO[Int] =
```

Comment:
Do we still need this functions? If not, please remove it.

Reply:
good catch, it wasn't needed